pkg/packagekit/updates.jsx

```diff
@@ -275,8 +275,8 @@ class UpdateItem extends React.Component {
         }
 
         var pkgList = this.props.pkgNames.map(n => (
-            <OverlayTrigger key={n} overlay={ <Tooltip id="tip-summary">{packageSummaries[n]}</Tooltip> } placement="top">
-                <span>{n}</span>
+            <OverlayTrigger key={n.name + n.arch} overlay={ <Tooltip id="tip-summary">{packageSummaries[n.name] + "(" + n.arch + ")"}</Tooltip> } placement="top">
```

Comment:
Please put a space in front of the (, so that this looks tasteful.

Reply:
I've added space